src/main/java/org/quiltmc/loader/impl/transformer/EnvironmentStrippingVisitor.java

```diff
@@ -36,9 +33,11 @@
 import net.fabricmc.api.EnvironmentInterface;
 import net.fabricmc.api.EnvironmentInterfaces;
 
+import org.quiltmc.loader.impl.util.StrippingDataContainer;
+
 /** Scans a class for Environment and EnvironmentInterface annotations to figure out what needs to be stripped. */
 @QuiltLoaderInternal(QuiltLoaderInternalType.LEGACY_EXPOSED)
-public class EnvironmentStrippingData extends ClassVisitor {
+public class EnvironmentStrippingVisitor extends ClassVisitor {
```

Comment:
Don't rename classes - especially those which are `LEGACY_EXPOSED` as mods can access them. I have no idea *why* mods might access/use this, but I'd prefer to *not* break mods unless absolutely necessary.